<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EasyNotesClass</class>
 <widget class="QDialog" name="EasyNotesClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>644</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>EasyNotes</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="margin">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="lblBasedOn">
          <property name="text">
           <string>Based on : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbBasedOn">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>User</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Case</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Machine</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblCaseList">
          <property name="text">
           <string>    Case list : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbCaseList">
          <property name="minimumSize">
           <size>
            <width>175</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>338</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnApply">
          <property name="text">
           <string>Apply</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="tblEasyNotes">
     <property name="minimumSize">
      <size>
       <width>591</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>465</height>
      </size>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <row>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../recorder.qrc">
        <normaloff>:/Recorder/Resources/attach.png</normaloff>:/Recorder/Resources/attach.png</iconset>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../recorder.qrc">
        <normaloff>:/Recorder/Resources/attach.png</normaloff>:/Recorder/Resources/attach.png</iconset>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../recorder.qrc">
        <normaloff>:/Recorder/Resources/attach.png</normaloff>:/Recorder/Resources/attach.png</iconset>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../recorder.qrc">
        <normaloff>:/Recorder/Resources/attach.png</normaloff>:/Recorder/Resources/attach.png</iconset>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../recorder.qrc">
        <normaloff>:/Recorder/Resources/attach.png</normaloff>:/Recorder/Resources/attach.png</iconset>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../recorder.qrc">
        <normaloff>:/Recorder/Resources/attach.png</normaloff>:/Recorder/Resources/attach.png</iconset>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../recorder.qrc">
        <normaloff>:/Recorder/Resources/attach.png</normaloff>:/Recorder/Resources/attach.png</iconset>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../recorder.qrc">
        <normaloff>:/Recorder/Resources/attach.png</normaloff>:/Recorder/Resources/attach.png</iconset>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../recorder.qrc">
        <normaloff>:/Recorder/Resources/attach.png</normaloff>:/Recorder/Resources/attach.png</iconset>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../recorder.qrc">
        <normaloff>:/Recorder/Resources/attach.png</normaloff>:/Recorder/Resources/attach.png</iconset>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../recorder.qrc">
        <normaloff>:/Recorder/Resources/attach.png</normaloff>:/Recorder/Resources/attach.png</iconset>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../recorder.qrc">
        <normaloff>:/Recorder/Resources/attach.png</normaloff>:/Recorder/Resources/attach.png</iconset>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../recorder.qrc">
        <normaloff>:/Recorder/Resources/attach.png</normaloff>:/Recorder/Resources/attach.png</iconset>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../recorder.qrc">
        <normaloff>:/Recorder/Resources/attach.png</normaloff>:/Recorder/Resources/attach.png</iconset>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../recorder.qrc">
        <normaloff>:/Recorder/Resources/attach.png</normaloff>:/Recorder/Resources/attach.png</iconset>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../recorder.qrc">
        <normaloff>:/Recorder/Resources/attach.png</normaloff>:/Recorder/Resources/attach.png</iconset>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../recorder.qrc">
        <normaloff>:/Recorder/Resources/attach.png</normaloff>:/Recorder/Resources/attach.png</iconset>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../recorder.qrc">
        <normaloff>:/Recorder/Resources/attach.png</normaloff>:/Recorder/Resources/attach.png</iconset>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../recorder.qrc">
        <normaloff>:/Recorder/Resources/attach.png</normaloff>:/Recorder/Resources/attach.png</iconset>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../recorder.qrc">
        <normaloff>:/Recorder/Resources/attach.png</normaloff>:/Recorder/Resources/attach.png</iconset>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Shortcut Key</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Shortcut Text</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Shortcut Label </string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>Alt + Ctrl + A</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>Alt + Ctrl + B</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string>Alt + Ctrl + C</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="3" column="0">
      <property name="text">
       <string>Alt + Ctrl + D</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="4" column="0">
      <property name="text">
       <string>Alt + Ctrl + E</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="5" column="0">
      <property name="text">
       <string>Alt + Ctrl + F</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="6" column="0">
      <property name="text">
       <string>Alt + Ctrl + G</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="7" column="0">
      <property name="text">
       <string>Alt + Ctrl + H</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="8" column="0">
      <property name="text">
       <string>Alt + Ctrl + I</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="9" column="0">
      <property name="text">
       <string>Alt + Ctrl + J</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="10" column="0">
      <property name="text">
       <string>Alt + Ctrl + K</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="11" column="0">
      <property name="text">
       <string>Alt + Ctrl + L</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="12" column="0">
      <property name="text">
       <string>Alt + Ctrl + M</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="13" column="0">
      <property name="text">
       <string>Alt + Ctrl + N</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="14" column="0">
      <property name="text">
       <string>Alt + Ctrl + O</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="15" column="0">
      <property name="text">
       <string>Alt + Ctrl + P</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="16" column="0">
      <property name="text">
       <string>Alt + Ctrl + Q</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="17" column="0">
      <property name="text">
       <string>Alt + Ctrl + R</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="18" column="0">
      <property name="text">
       <string>Alt + Ctrl + S</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="19" column="0">
      <property name="text">
       <string>Alt + Ctrl + T</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEnabled</set>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../recorder.qrc"/>
 </resources>
 <connections/>
</ui>
